build_info/hicolor-icon-theme.prerm

```diff
@@ -0,0 +1,8 @@
+#! /bin/sh
+set -e
+
+if [ "$1" = "remove" ] || [ "$1" = "deconfigure" ] ; then
+  rm -f /usr/share/icons/hicolor/icon-theme.cache
```

Comment:
Use `@MEMO_PREFIX@@MEMO_SUB_PREFIX@`, and add the postinst and triggers from Debian.

Reply:
We don’t have gtk yet to which is why I’m waiting on that. If you want

Reply:
I wouldn’t wait, it won’t fail if missing gtk, so might as well add it so that we don’t need future changes down the line.

Reply:
I’ll start working on getting gtk uploaded this week(without gir for now)

Reply:
Don't worry about getting gtk added before adding the trigger and postinst.

Reply:
Na I'm gonna add the trigger and postinst. It's just that I have nothing to do for basically the whole week.